src/Google/Client.php

```diff
@@ -795,7 +796,14 @@ public function execute(RequestInterface $request, $expectedClass = null)
     // this is where most of the grunt work is done
     $http = $this->authorize();
 
-    return Google_Http_REST::execute($http, $request, $expectedClass, $this->config['retry']);
+    return Google_Http_REST::execute(
+        $http,
+        $request,
+        $expectedClass,
+        $this->config['retry'],
+        $this->config['retry_map']
+    );
+    
```

Comment:
Could we please drop the whitespace here? LGTM otherwise.

Reply:
done